src/core/ext/filters/client_channel/lb_policy/xds/xds_cluster_impl.cc

```diff
@@ -264,6 +266,58 @@ class XdsClusterImplLb : public LoadBalancingPolicy {
   RefCountedPtr<RefCountedPicker> picker_;
 };
 
+//
+// XdsClusterImplLb::Picker::SubchannelCallTracker
+//
+
+class XdsClusterImplLb::Picker::SubchannelCallTracker
+    : public LoadBalancingPolicy::SubchannelCallTrackerInterface {
+ public:
+  SubchannelCallTracker(
```

Comment:
can we maybe add a debug mode assert in the destructor to make sure that a `Start()` is always accompanied by a `Finish()`?

Reply:
Done.